compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/symbols/TupleTypeSymbol.java

```diff
@@ -33,6 +33,13 @@ public interface TupleTypeSymbol extends TypeSymbol {
      */
     List<TypeSymbol> memberTypeDescriptors();
 
+    /**
+     * Get the tuple member types.
+     *
+     * @return {@link List} of member types
+     */
+    List<TupleMemberTypeSymbol> tupleTypeMembers();
```

Comment:
This again is tuple members, right? IMO, can just be

```java
    List<TupleMemberTypeSymbol> members();
```

or if we need to include tuple

```java
    List<TupleMemberTypeSymbol> tupleMembers();
```

Docs need to be updated too.

Reply:
+1 and please check my thoughts on https://github.com/ushirask/ballerina-lang/pull/54#discussion_r1062132733